include/roboteam_ai/control/NewControlUtils.h

```diff
@@ -0,0 +1,105 @@
+//
+// Created by timovdk on 2/21/20.
+//
+
+#ifndef ROBOTEAM_AI_NEWCONTROLUTILS_H
+#define ROBOTEAM_AI_NEWCONTROLUTILS_H
+
+#include <roboteam_utils/Arc.h>
+#include <roboteam_utils/Line.h>
+#include <cmath>
+#include <optional>
+#include "utilities/Constants.h"
+#include "world_new/FieldComputations.hpp"
+
+using Vector2 = rtt::Vector2;
+using Angle = rtt::Angle;
+
+namespace rtt::ai::control {
+
+    using namespace rtt::ai::world;
+
+    class NewControlUtils {
```

Comment:
We have decided that it is very important to document all code that you add (with Javadoc), which means that at least:
- Every class should have a short summary of what it does and includes the tag "author" and "since".
- Every public function should have a short summary of what it does, what the meaning is of all parameters and what it returns (if it returns something) and which exceptions it might throw (if the function throws an exception).